README.md

```diff
@@ -1,2 +1,311 @@
 # logagg
 logs aggregation framework
+
+Collect all the logs from server and parses it by making common schema for all the logs and stores at given storage engine.
+
+## Installation
+> Prerequisites: Python2.7
+> Note: tested and it works in python2.7 and not tested in other versions.
+
+- Install the `logagg` package by running commands,
+```bash
+git clone "https://github.com/deep-compute/logagg.git"
+cd logagg
+sudo python setup.py install
+```
+
+## Usage
+
+### Bring up the `nsq` instances at the server
+```
+nsqlookupd
+nsqd -lookupd-tcp-address localhost:4160
+nsqadmin -lookupd-http-address localhost:4161
+```
+
+### Types of handlers we support
+- nginx_access
+- django
+- mongodb
+- elasticsearch
+- basescript
+
+### Getting help per command
```

Comment:
@praneeth36 @AnushaKonduru We can how something like `Help command` as this section title.